recipes/jigsaw/build.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+cd src
+
+${GXX} -std=c++11 -pedantic -Wall -s -O3 -flto -D NDEBUG -D __cmd_jigsaw \
```

Comment:
Use `${CC}` and `${CXX}` instead.

Reply:
Okay, I'll update these now.

Reply:
Do you think this is related to the OSX failures?  Or do I just have incompatible build flags here or something like that?

Reply:
It does look like these could be the problem, since ${GXX} seems to be undefined on OSX.

Reply:
Yeah, on macOS you get `${CLANG}` and `${CLANGXX}` instead but it's better to use the generic variables rather than assume macOS uses clang and Linux uses GCC.